site/ja/guide/function.ipynb

```diff
@@ -486,16 +485,61 @@
         "v"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "mxjNpA6Ua4_o"
+      },
+      "source": [
+        "<a id=\"debugging\"></a>\n",
+        "\n",
+        "## デバッグ\n",
+        "\n",
+        "`tf.function` や AutoGraph はコードを生成し、それを TensorFlow のグラフにトレースします。この機構は現在のところ、`pdb` のようなステップ・バイ・ステップのデバッガーをサポートしていません。ですが、`tf.config.experimental_run_functions_eagerly(True)` を呼び出すことで、一時的に Eager Execution を `tf.function` の中で有効にして、好きなデバッガーを用いることができます。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "i1HtHX6Ja4_o"
+      },
+      "outputs": [],
+      "source": [
+        "@tf.function\n",
+        "def f(x):\n",
+        "  if x > 0:\n",
+        "    # Try setting a breakpoint here!\n",
+        "    # Example:\n",
+        "    #   import pdb\n",
+        "    #   pdb.set_trace()\n",
+        "    x = x + 1\n",
+        "  return x\n",
+        "\n",
+        "tf.config.experimental_run_functions_eagerly(True)\n",
+        "\n",
+        "# You can now set breakpoints and run the code in a debugger.\n",
```

Comment:
Do we need to translate this into Japanese?

Reply:
Same as above, I translate this comment.